object/user.go

```diff
@@ -30,7 +30,7 @@ type User struct {
 	Password      string `xorm:"varchar(100)" json:"password"`
 	PasswordType  string `xorm:"varchar(100)" json:"passwordType"`
 	DisplayName   string `xorm:"varchar(100)" json:"displayName"`
-	Avatar        string `xorm:"varchar(100)" json:"avatar"`
+	Avatar        string `xorm:"varchar(1000)" json:"avatar"`
```

Comment:
Why, 100 isn't enough for image link?

Reply:
100 is not enough for image link. For example, casdoor deployed at https://door-hk.chromium.link, has an avatar link: `https://aliyun-wb-bl6qnlpj4m.oss-cn-shanghai.aliyuncs.com/casdoor/avatar/admin.png?time=1615906437522268542`, it has 107 characters.